privaterelay/settings.py

```diff
@@ -437,6 +437,8 @@ def _get_initial_middleware() -> list[str]:
 
 MAX_ADDRESS_CREATION_PER_DAY = config("MAX_ADDRESS_CREATION_PER_DAY", 100, cast=int)
 MAX_REPLIES_PER_DAY = config("MAX_REPLIES_PER_DAY", 100, cast=int)
+MAX_FORWARDED_PER_DAY = config("MAX_FORWARDED_PER_DAY", 1000, cast=int)
+MAX_FORWARDED_EMAIL_SIZE_PER_DAY = config("MAX_FORWARDED_EMAIL_SIZE_PER_DAY", 1000_000_000, cast=int)
```

Comment:
Nit: once more underscore
```suggestion
MAX_FORWARDED_EMAIL_SIZE_PER_DAY = config("MAX_FORWARDED_EMAIL_SIZE_PER_DAY", 1_000_000_000, cast=int)
```